src/index.js

```diff
@@ -1,33 +1,76 @@
 import './style.css';
 
 const form = document.querySelector('form');
+const inputEl = document.querySelector('.input');
 const todoContainer = document.querySelector('.todoContainer');
 
-const lists = [{
-  description: 'Start being me',
-  bool: 'false',
-  index: '1',
-},
-{
-  description: 'Just be happy',
-  bool: 'false',
-  index: '2',
-}, {
-  description: 'I am kenny',
-  bool: 'false',
-  index: '3',
-}];
+let todo;
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+const store = () => {
+  todo = {
+    Description: inputEl.value,
+    id: todos.length,
+    completed: false,
+  };
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
 
-lists.forEach((task) => {
-  form.addEventListener('submit', (e) => {
-    e.preventDefault();
-    const ul = document.createElement('ul');
-    const hr = document.createElement('hr');
-    ul.innerHTML = `
-    <li>
-    <input class="checkBox" type="checkbox">${task.description}<i class="fa-solid fa-ellipsis-vertical dots"></i>
-    </li>
-    `;
-    todoContainer.append(ul, hr);
+const clear = () => {
+  inputEl.value = '';
+};
+
+const removeBook = (id) => {
```

Comment:
- [ ] As required by the project requirements, your removeBook function should be added to a new Javascript file.